examples/nirfmxnr/acp-single-carrier.py

```diff
@@ -0,0 +1,309 @@
+# Steps:
+# 1. Open a new RFmx Session.
+# 2. Configure Frequency Reference.
+# 3. Configure Selected Ports.
+# 4. Configure basic signal properties (Center Frequency, RF Attenuation and External Attenuation).
+# 5. Configure Trigger Parameters for IQ Power Edge Trigger.
+# 6. Configure Link Direction, Frequency Range and Carrier Bandwidth and Subcarrier Spacing.
+# 7. Configure Reference Level.
+# 8. Select ACP measurement and enable Traces.
+# 9. Configure Measurement Method.
+# 10. Configure Noise Compensation Parameter.
+# 11. Configure Sweep Time Parameters.
+# 12. Configure Averaging Parameters for ACP measurement.
+# 13. Initiate the Measurement.
+# 14. Fetch ACP Measurements and Traces.
+# 15. Close RFmx Session.
+
+import sys
+
+import grpc
+import nirfmxnr_pb2 as nirfmxnr_types
+import nirfmxnr_pb2_grpc as grpc_nirfmxnr
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "NI-RFSASession"
```

Comment:
we should change up these throwaway names to avoid RFSA (also "SimulatedRFSA" below)

Good options:
* `RFmxNRSession`
* `ACPSingleCarrierSession`
* `SimulatedDevice`
* `Simulated5663`

Different string-casing conventions are also fine.